types/whatwg-url/ts3.9/test/whatwg-url.webidl2js.test.ts

```diff
@@ -0,0 +1,88 @@
+import * as whatwgUrl from "whatwg-url";
+// tslint:disable-next-line: no-relative-import-in-test
+import { URL, URLSearchParams } from "../webidl2js-wrapper";
+
+// `import { URL, URLSearchParams } from "whatwg-url/webidl2js-wrapper";`
+// fails without `"paths": { "whatwg-url": ["whatwg-url/ts3.9"] }` in tsconfig.json,
+// but that causes @definitelytyped/definitions-parser to complain.
```

Comment:
@orta @sandersn @weswigham This looks like the same error as in <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50541#pullrequestreview-565815110>.